.github/workflows/build.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         with:
           ROS_DISTRO: ${{ matrix.ros }}
           DEB_DISTRO: ${{ matrix.deb }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
You have to remove this commit again. I added this with the `[DBG]` tag so that the CI can run through. But this disables publishing the Debian packages. I mean, you could also just leave it as is to just check the Debian package building, but then I would rename the commit to make it clear that this is to not publish the packages.

Reply:
Changed in https://github.com/AndrejOrsula/pymoveit2/pull/97/commits/f883c366bf275a3e975b3dc37fe614e17e02537a